taccsite_cms/static/site_cms/js/modules/setTargetForExternalLinks.js

```diff
@@ -1,37 +1,4 @@
-/**
- * Whether to log debug info to console
- * @const {string}
- */
-const SHOULD_DEBUG = window.DEBUG;
-
-/**
- * Function to perform after setting link target
- * @callback setTargetCallback
- * @param {HTMLElement} link
- */
-
-/**
- * Set external links (automatically discovered) to open in new tab
- * @param {object} [options] - Optional parameters
- * @param {array.<string>} [options.pathsToExernalSite=[]] - (DEPRECATED) A list of relative URL paths that should be treated like external URLs
- * @param {array.<string|RegExp>} [options.pathsToForceSetTarget=[]] - A list of relative URL paths (or patterns) that should trigger setting a target
- * @param {HTMLElement|Document} [options.scopeElement=document] - The element within which to search for links
- * @param {setTargetCallback} [options.setTargetCallback] - A callback for after a target is set
- */
```

Comment:
**Please retain the [docblock](https://en.wikipedia.org/wiki/Docblock) sans `@param` lines.** The function description is still accurate.

<sup>Docblocks are written in [jsDoc format](https://jsdoc.app/about-block-inline-tags). They provide manual definitions (in lieu of TypeScript), and can be used to auto-generate documentation e.g. [vite.js on jsdocs.io](https://www.jsdocs.io/package/vite#RollupCommonJSOptions.transformMixedEsModules).</sup>

Reply:
Restored via [cbad538](https://github.com/TACC/Core-CMS/pull/805/commits/cbad538bc59e333040f84fd3e665cb388c71b5af) without params.